compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BJSONType.java

```diff
@@ -85,4 +86,13 @@ public void accept(TypeVisitor visitor) {
     public <T, R> R accept(BTypeVisitor<T, R> visitor, T t) {
         return visitor.visit(this, t);
     }
+
+    @Override
+    public SemType semType() {
+        // Ideally this shouldn't be needed but somehow in nBallerina code base, there was an edge case where members
```

Comment:
Do you mean jBallerina code base?
```suggestion
        // Ideally this shouldn't be needed but somehow in jBallerina code base, there was an edge case where members
```